### ISSUE TITLE
[Bug]: Code Generation doesn't respect Model's dedicated Eloquent Builder

### ISSUE THREAD

**Issue by cosmastech** [laravel-idea/plugin#735]
### Bug description

In our application, we have some code like this:

```php
namespace App\Models;

abstract class BaseModel extends Illuminate\Database\Eloquent\Model
{
    public function newEloquentBuilder($query): App\Database\Eloquent\BaseEloquentBuilder
    {
        return new App\Database\Eloquent\BaseEloquentBuilder($query);
    }
}
```

```php
namespace App\Database\Eloquent;

class BaseEloquentBuilder extends Builder
{
    /**
     * Add an "order by" clause to the query.
     *
     * @param  Closure|QueryBuilder|Builder|Expression|string|array $column
     * @param  string  $direction
     * @return $this
     *
     * @throws InvalidArgumentException
     */
    public function orderBy($column, $direction = 'asc'): static
    {
        // handle case where is_array($column)
        return self::orderBy($column, $direction);
    }
}
```

All of our application's models extend from BaseModel and all Model::query() should be referencing `BaseEloquentBuilder`

When I `Generate Helper Code` the helper files still reference Illuminate\Database\Query\Builder instead of our application's BaseEloquentBuilder.

### Plugin version

6.3.1.223

### Operating system

MacOS

### Steps to reproduce

1. Spin up a Laravel project
2. Create a BaseModel class (like described above)
3. Create a BaseEloquentBuilder class (like described above)
4. Modify BaseModel@newEloquentBuilder() to return BaseEloquentBuilder
5. Create a model (MyModel) that extends from BaseModel
6. In a controller, write `MyModel::query()->orderBy([])`
<img width="749" alt="image" src="https://user-images.githubusercontent.com/42181698/223497208-ff49e3e7-7769-4370-a36c-581b12a468da.png">

**Comment by adelf** [comment]
Yes. Laravel Idea searches the newEloquentBuilder method only inside the model class... I'll try to fix that. Thanks.

**Comment by adelf** [comment]
Fixed in 7.0
